src/TurbulenceConvection/types.jl

```diff
@@ -253,91 +197,22 @@ function Base.summary(io::IO, edmf::EDMFModel)
 end
 
 
-struct State{P, A, TS, T, CACHE, C}
+struct State{P, A, T, CACHE, C}
     prog::P
     aux::A
-    ts_sfc::TS
     tendencies::T
     p::CACHE
     colidx::C
+    surface_conditions::Any
```

Comment:
```suggestion
    surface_conditions::SC
```
Add `SC` to the type parameters of `State` so that the compiler knows the type of this field. This will reduce allocations.